launcher/internal/systemctl/systemctl.go

```diff
@@ -0,0 +1,62 @@
+// Package systemctl implements a subset of systemctl operations.
+package systemctl
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/coreos/go-systemd/dbus"
+)
+
+// Systemd is an interface to connect to host systemd with selected functions.
+type Systemd interface {
+	Start(string) error
+	Stop(string) error
+	Close()
+}
+
+// Systemctl is a wrap around of dbus.Conn and implements the Systemd interface.
+type Systemctl struct {
+	dbus *dbus.Conn
+}
+
+var _ Systemd = (*Systemctl)(nil)
+
+// New connects to systemd over dbus.
+func New() (*Systemctl, error) {
+	conn, err := dbus.New()
+	if err != nil {
+		return nil, err
+	}
+	return &Systemctl{dbus: conn}, nil
+}
+
+// Start is the equivalent of `systemctl start $unit`.
+func (s *Systemctl) Start(unit string) error {
+	return runSystemdCmd(s.dbus.StartUnit, "start", unit)
```

Comment:
Seems like StartUnit is deprecated, try to use StartUnitContext instead https://github.com/coreos/go-systemd/blob/v22.5.0/dbus/methods.go#L116, (same for the following functions)

It's probably because you imported a old version of dbus